pandapipes/toolbox.py

```diff
@@ -615,3 +615,6 @@ def get_internal_tables_pandas(net, convert_types=True):
                         tbl[col] = tbl[col].astype(np.bool_)
 
     return node_table, branch_table
+
+def calculate_alpha(d, u):
```

Comment:
I would rename this function and signature: def alpha_from_u(u, s) . As far as I can recall using s for the wall thickness is more common than d, which is usually the diameter. Otherwise, it could make sense to name it precisely as thickness.

Reply:
And to avoid confusion, I would write some docs, describing the meaning of alpha, u and s.